### PR TITLE
Upgrading to ReactNative 0.53.3

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
`Android` and `iOS` apps are running fine.

**Comment by bhariharan** [comment]
@wmathurin Should I get rid of all the `test` stuff we have here, now that we have proper bridge tests in the source repos?

File: installios.js [hunk 1]
```diff
@@ -22,4 +22,4 @@ console.log('Installing pod dependencies');
 execSync('pod update', {stdio:[0,1,2], cwd:'ios'});
 
 console.log('Generating main.jsbundle for tests');
-execSync('react-native bundle --platform ios --dev false --entry-file index.js --bundle-output IntegrationTests/main.jsbundle', {stdio:[0,1,2]});
+execSync('node_modules/.bin/react-native bundle --platform ios --dev false --entry-file index.js --bundle-output IntegrationTests/main.jsbundle', {stdio:[0,1,2]});
```
Comment (wmathurin) [inline installios.js]: 👍 
I think I'm going to get rid of those tests now that we have real tests in the SDK.

**Review by wmathurin — APPROVED** [review]

File: installios.js (hunk 1 above)
Comment (wmathurin) [reply installios.js]: I didn't see your comment above.
Yeah you can go ahead and clean them up if you want.

Comment (wmathurin) [reply installios.js]: There is stuff in the podfile https://github.com/forcedotcom/SmartSyncExplorerReactNative/blob/master/ios/Podfile#L41, a couple of test folder, maybe some stuff on the android side as well.

**Comment by bhariharan** [comment]
@wmathurin Removed tests [here](https://github.com/forcedotcom/SmartSyncExplorerReactNative/pull/143/commits/0f67297acbcf12ec8fbb89f0017843c567e384da).

**Review by wmathurin — APPROVED** [review]

**Pull request merged**